src/test/java/bio/terra/workspace/common/BaseConnectedTest.java

```diff
@@ -11,4 +12,5 @@
  */
 @Tag("connected")
 @ActiveProfiles("connected-test")
+@AutoConfigureMockMvc
```

Comment:
Are we expecting to use MockMvc in connected other connected tests or is this just for WorkspaceServiceTest? If the latter, the annotation should be there (modulo keeping that as a connected test)

Reply:
good point, done.